schemas/rule.yml

```diff
@@ -0,0 +1,84 @@
+---
+- name: rule
+  title: Rule
+  group: 2
+  short: Fields to capture details about rules used to generate alerts or other events.
+  description: >
+   Rule fields are used to capture the specifics of any observer or agent rules that generate alerts or other events.
```

Comment:
I'm good with that, will update today

on the other question - rule.category is, typically, the category of action the rule is alerting on (e.g. "Web Attacks" rulesets would include xss, directory traversal, etc) and you could then apply the "WEB ATTACKS" rule wherever it needed to be applied, instead of enabling each individual rule.  

ruleset is a logical configuration which (often times across multiple managed implementations of a solution - e.g. Panorama managing multiple Palo Altos) combines rules into a single logical set (e.g. on the GUEST NETWORK apply the GUEST rule.ruleset which includes [blocks DNS-over_TLS, blocks nsfw/guns/and competitors web sites,  limits bandwidth, etc.)

Reply:
So ruleset is a wider grouping than category? Or are these orthogonal?

Reply:
orthogonal...

Ruleset is localized configuration/implementation specific (as in apply ruleset MY_RULES to this subnet / vlan / group) where MY_RULES have been defined to include e.g. 10 specific locally relevant individual rules.   

Category is a grouping of rules by type (think Mitre technique or vulnerable system type)